src/index.ts

```diff
@@ -266,20 +271,21 @@ export default class MachinomyServerPlugin extends MiniAccountsPlugin {
       account.balance = newBalance
       this._log.trace(`account ${account.accountName} debited ${formatAmount(amount, 'eth')}, new balance ${formatAmount(newBalance, 'eth')}`)
 
+      // Expiry timer for reject
+      let timer = new Promise(resolve => setTimeout(() => {
```

Comment:
make sure to clear this timer after the Promise.race

Reply:
Done.